weasyl/test/test_resetpassword.py

```diff
@@ -107,31 +109,35 @@ def testPrepare_VerifyDeletionOfStaleRecordsOlderThanOneHour():
         user_name = TestFunctions().generateTestAccountName()
         user_id = db_utils.create_user(username=user_name)
         email_addr = user_name + "@weasyl.com"
-        d.execute("UPDATE login SET email = '%s' WHERE userid = %i", [email_addr, user_id])
+        d.engine.execute("UPDATE login SET email = %(email)s WHERE userid = %(id)s", email=email_addr, id=user_id)
         form_for_request = Bag(email=email_addr, username=user_name, day=arrow.now().day,
                    month=arrow.now().month, year=arrow.now().year)
         # Emails fail in test environments
         with pytest.raises(OSError) as err:
             resetpassword.request(form_for_request)
         assert 'OSError' in str(err)
-        query = d.execute("SELECT token FROM forgotpassword WHERE userid = %i", [user_id])
-        token = str(query[0][0])
-        token_store.append(token)
+        query = d.engine.execute("SELECT token FROM forgotpassword WHERE userid = %(id)s", id=user_id)
+        token = query.first()[0]
+        token_store.append(str(token))
```

Comment:
Is the cast to `str` here because `token` is a `unicode`? `token.encode('utf-8')` might be more explicit if so.